File: presets/MidiKit/JavaScript/PassThrough.js
/**
 * @target stoermelder MIDI-KIT
 * @engine QuickJs@v1
 * @author stoermelder
 * @description Passes all incoming MIDI messages to the default MIDI output port.
 */

midi.onMessage = function(midiPort, msg) {
    midiOut.send(msg);
}

File: presets/MidiKit/Lua/PassThrough.lua
--[[
@target stoermelder MIDI-KIT
@engine minilua@v1
@author stoermelder
@description Passes all incoming MIDI messages to the default MIDI output port.
--]]

rack.onMidiMessage = function(midiPort, msg)
    midiOut.send(msg)
end